types/mac.go

```diff
@@ -119,6 +119,17 @@ func (a MacArray) String() string {
 	return o.String()
 }
 
+func (a *MacArray) ValueString(i int) string {
+	arr := a.Storage().(*array.Binary)
+	switch {
+	case a.IsNull(i):
+		return "(null)"
+	default:
+		mac := net.HardwareAddr(arr.Value(i))
+		return fmt.Sprintf("\"%s\"", mac)
```

Comment:
```suggestion
		return fmt.Sprintf("%q", mac)
```
A macaddr shouldn't need it, but just for consistency